index.js

```diff
@@ -8,6 +8,7 @@ var extname = require('path').extname;
 var basename = require('path').basename;
 var resolve = require('path').resolve;
 var testit = require('testit');
+var camelCase = require('camelcase')
```

Comment:
Im not agree. dirname, pathname, basename are always written in lowercase, it is convention and because they are like that in the node internally. `testit` not make sense to be `testIt`.